### DISCUSSION TITLE
imputation with machine learning

### DISCUSSION THREAD

**Discussion by mehrdad123qaderi** [amices/mice#652] [Missing data methodology]
Regards, I need a source that theoretically explains the machine learning methods used for imputation of missing data with full explanation.

**Comment by hanneoberman** [comment]
This chapter provides an overview of different imputation methods: https://dspace.library.uu.nl/handle/1874/436408

**Comment by stephematician** [comment]
You would likely get more responses on <https://stats.stackexchange.com/>. That's more of a 'community forum' for people in machine learning than github ever will be.
